task/options/strconv.go

```diff
@@ -0,0 +1,109 @@
+package options
+
+import (
+	"fmt"
+	"strconv"
+	"unicode"
+	"unicode/utf8"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/codes"
+)
+
+// TODO(jsternberg): This file copies over code from an internal package
+// because we need them from an internal package and the only way they
+// are exposed is through a package that depends on the core flux parser.
+// We want to avoid a dependency on the core parser so we copy these
+// implementations.
+//
+// In the future, we should consider exposing these functions from flux
+// in a non-internal package outside of the parser package.
```

Comment:
Yeah, I second this. That was my first thought when I saw this file and the other duplicated `ParseDuration` function above. Do we have an issue for this?

Reply:
No I'll make one.

Reply:
https://github.com/influxdata/flux/issues/2588